packages/blocks/src/api/factory.js

```diff
@@ -88,8 +88,38 @@ export function createBlocksFromInnerBlocksTemplate(
 }
 
 /**
- * Given a block object, returns a copy of the block object, optionally merging
- * new attributes and/or replacing its inner blocks.
+ * Given a block object, returns a copy of the block object without sanitizing its attributes,
+ * optionally merging new attributes and/or replacing its inner blocks.
+ *
+ * @param {Object} block              Block instance.
+ * @param {Object} mergeAttributes    Block attributes.
+ * @param {?Array} newInnerBlocks     Nested blocks.
+ *
+ * @return {Object} A cloned block.
+ */
+export function duplicateBlock( block, mergeAttributes = {}, newInnerBlocks ) {
```

Comment:
What should be a better name for this? `cloneBlockWithoutSanitizingAttributes`?

Also, is it possible to mark this method private only for now? Should we?

Reply:
Making it `__experimental` (not for public use and liable to change) is the closest equivalent.

Alternatively there's `__unstable` which is used for anything temporary.

As an aside, I'm not sure sanitize is the right word, as to me this makes it sound like the values of the attributes will be cleaned. But maybe it's too difficult to change that now. (it'd have to be patched into WordPress 5.7 before release, and deprecated in the plugin)

I'd maybe consider something like `cloneWithExtraneousAttributes`, and the sanitize function could be `omitExtraneousAttributes` or `withoutExtraneousAttributes`.